src/retrace-server-bugzilla-query

```diff
@@ -7,6 +7,7 @@ to a log and for tasks found in the comments set bugzillano field with bugzilla
 import sys
 import re
 import time
+from datetime import datetime, timedelta
 
```

Comment:
Drop this empty line please.

Reply:
Yes I thought about making those parameters and I think you're right.  These settings are related to the DeleteTaskAfter and DeleteFailedTaskAfter too, as well as the cronjob settings for the various retrace-server-* tools.  So it's a bit of a complicated setup and probably they should all be configurable.

I also thought about the WARNING in there - probably that's not ideal.  It would be better to detect the limit is hit then run another query, assuming the API has a way to pickup where you left off - maybe start with a specific bug# and only ask for bugs > bug#?  I'll need to look a bit more at the bugzilla API.